Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Build controller-manager
+ARG GO_VERSION
+FROM golang:$GO_VERSION AS builder
+
+WORKDIR $GOPATH/src/github.com/k8snetworkplumbingwg/kubemacpool
+COPY . .
```

Comment:
better copy just the relevant files please

Reply:
I need all the project in order to build the project.
The irrelevant folders like _kubevirtci is on the Dockerignore file.